integration_tests/pyproject.toml

```diff
@@ -20,10 +20,11 @@ python-dateutil = "^2.8.1"
 web3 = "^5.20.1"
 eth-bloom = "^1.0.4"
 python-dotenv = "^0.19.2"
-pystarport = "^0.2.3"
+pystarport = { git = "https://github.com/mmsqe/pystarport.git", branch = "jsonnet-config-simplify" }
```

Comment:
can use main branch now.

Reply:
yes, but not sure if ci can cache with git url since current build take ages

Reply:
https://github.com/crypto-org-chain/cronos/pull/558/files#diff-b9a0014cf6c4615d1c2e8d921dc3db59b4a68b462788d667ed622331660d17cdR69
that's becuase you updated `nixpkgs`, you can revert those changes, then it'll be much more cache friendlier.